site/content/privacy-policy/index.html

```diff
@@ -15,26 +15,6 @@ <h2> General website information</h2><p>
 <br>
   We share this information only with our website hosting service GitHub Pages, who are under an obligation not to use your information for any other purpose.
 </p>
-  <h2> Cookies</h2>
-  <p>
-  To make this website work properly, we sometimes place small data files called “cookies” on your device. Most big websites do this too.</p>
-  <h3>What are cookies?</h3>
-  <p>A cookie is a small text file that a website (including this website) saves on your computer or mobile device when you visit the site. It enables the website, including plugins such as YouTube, to remember your actions and preferences (such as login, language, font size and other display preferences) over a period of time, so you don’t have to keep re-entering them whenever you come back to the site or browse from one page to another. It also enables us to analyse how this website is being used.</P>
-  <h3> How do we use cookies?</h3>
-<p>  We use Google Analytics to help us understand how people are using our website. The following lines explain the cookies we use and why we use them.
-<br>
-  Cookies called <code>_ga</code> and <code>_gid</code> are used by Google Analytics to distinguish users.
-<br>
-  A cookie called <code>_gat</code> is used by Google Analytics to throttle the request rate.
-
-
-</P>
-  <p>For more information see the <a href="https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage">Google Analytics cookies page</a>.
-
-</P>
-  <h3> How to control cookies</h3>
-<p>  You can control and/or delete cookies as you wish – for details, see <a href="aboutcookies.org">aboutcookies.org</a>. You can delete all cookies that are already on your computer and you can set most browsers to prevent them from being placed. If you do this, however, you may have to manually adjust some preferences every time you visit a site and some services and functionalities may not work.
-</P>
```

Comment:
Flagging this for @mundaym .

It is probably fine to remove, but James and I weren't sure. There aren't any cookies used by the site anymore. There will of course be cookies on the client until they expire.

Reply:
LGTM, thanks both